src/components/button5.tsx

```diff
@@ -1,5 +1,41 @@
-export default function Button5() {
-    return (
-        <button></button>
-    );
+'use client';
+import { useState } from "react";
+
+interface prop {
+    label: string;
 }
+
+
+export default function Button5(props: prop) {
+    const [weather, setWeather] = useState<{probabilityOfPrecipitation: string[]} | null>(null);
+
+    const getWeather = () => {
+        fetch(`https://api.weather.gov/gridpoints/BOX/69,92/forecast`)
+        .then(res => res.json())
+        .then(res => {
+            setWeather(res.properties.periods[0])
+        }); 
+    };
+    
+    return (
+        <div>
+            <button onClick={getWeather} className="bg-red-700 border-[6px] border-red-900  text-white px-12 py-4 rounded-full text-3xl font-bold">{props.label}</button>
+
+            {weather ? (
+                weather.probabilityOfPrecipitation.value > 55 ? (
+                    <div>
+                        <p className="text-center font-bold">Dream harder. We are in New England. {weather.probabilityOfPrecipitation.value}% chance of rain today.</p>
```

Comment:
My system is getting angry at the `weather.probabilityOfPrecipitation.value` line, saying that `weather.probabilityOfPrecipitation` doesn't have a member `value`. It is declared as a `string[]` on line 10, after all. 

I also am not seeing any number on the button, just the percent sign.

Maybe worth taking another look? Make sure you are running this with our version of node? It is also possible there is an issue on my end

Reply:
Ah i looked a little closer, it looks like the `value` could be null, which needs to be handled specially.